src/main/webapp/app/entities/iris/iris.model.ts

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import { Exercise } from 'app/entities/exercise.model';
+import { User } from 'app/core/user/user.model';
+
+export class IrisConversation {
+    id: number;
+    programmingExercise?: Exercise;
+    user?: User;
+    latestMessageId?: string;
+    messageDescriptors: IrisMessageDescriptor[];
```

Comment:
Needs to be the same as on the server.
```suggestion
    messages: IrisMessageDescriptor[];
```